### DISCUSSION TITLE
Any feature-rich DataGrid?  Or: Uno Telerik UI for UWP - anybody got it working?

### DISCUSSION THREAD

**Discussion by qfever** [unoplatform/uno#11965] [General]
Hello guys!
I am planning to port our ERP/CRM to Uno in the next 2 years and currently trying to find out gaps and building a "framework" for us (like MDI, etc.).

So there is a big need for a feature rich DataGrid/GridView for us, with built in dynamic column-filtering, conditional formatting, etc - currently we using WPF and Teleriks RadGridView, which is fitting our needs perfectly. I already had a look at WCT DataGrid (currently in my test-project) but unfortunately this is too "raw" for us - so I am trying to get the [Uno Telerik UI For UWP](https://github.com/unoplatform/Uno.Telerik.UI-For-UWP) running, but got some errors here, and I think its not compatible to newest Uno Platform, right?

So my question: Are there any people here with similar requirements for such a DataGrid/GridView and maybe want to develop one together? In further steps I also will need a Scheduler/Gantt.

Best regards

**Comment by sasakrsmanovic** [comment]
Hello 

There are a few options you have

- Send a request directly to Telerik for their commercial one. Please use this link to vote and post a comment. You can just paste your above comments. https://feedback.telerik.com/winui/1499061-add-support-for-uno-platform . Every vote helps; Telerik is known to listen to their customer community (full disclosure, I used to work for Telerik for 5 years) . 

- Another effort is underway to develop a 'datagrid light' as part of Windows Community Toolkit - https://github.com/CommunityToolkit/Labs-Windows/discussions/415#discussioncomment-5589485 . If you are interested

- Given the long term project and your will to invest in developing a grid, I also think there are ways to make WCT Grid work for you; we have many clients which have great results. Perhaps we could contribute to the WCT Grid too. I'd love to learn more about your project - please reach out to sasha at platform.uno if you are interested to chat.

**Reply by qfever** [reply]
Thank you very much! I wrote you via discord, maybe you wanna chat there :)
